client/src/components/pages/create-feed/Step2/index.tsx

```diff
@@ -79,8 +80,16 @@ const Step2 = (props: Step2Props): JSX.Element => {
 							alt="Title Image"
 						/>
 					</Descriptions.Item>
-					<Descriptions.Item label="Category">
-						{ readableCategoryValue }
+					<Descriptions.Item label="Tags">
+						{readableTagValue.map(tag => {
+							return (
+								<div key={tag} className={'custom-tag'} style={{ backgroundColor: stringToColor(tag) }}>
+									<Typography.Text style={{ color: (parseInt(stringToColor(tag).replace('#', ''), 16) > 0xffffff / 2) ? '#000' : '#fff', background: (parseInt(stringToColor(tag).replace('#', ''), 16) > 0xffffff / 2) ? '#fff' : '#000', opacity: 0.9 }}>
```

Comment:
Use custom method to gives better context & cleaner code. Since it's repeated, consider to move into common method or adding an layer of abstract.